applications/fastapi-bootcamp/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+appVersion: 0.1.0
```

Comment:
```suggestion
appVersion: "tickets-DM-43967
```

Reply:
Are we going to leave it on the ticket branch?

Reply:
I guess since I don't know the released version I'd have to change it here sometime anyway, so...sure.